examples/data-objects/codemirror/src/codemirror.ts

```diff
@@ -229,11 +229,11 @@ export class CodeMirrorComponent
     }
 
     public async request(request: IRequest): Promise<IResponse> {
-        return {
-            mimeType: "fluid/object",
-            status: 200,
-            value: this,
-        };
+        if (request.url === "" || request.url === "/") {
```

Comment:
Instead of duplicating this code around (and potentially having a bug and needing again to fix all the places and potentially missing some), can we have a helper function and using everywhere?